Resources/GameData/kOS/Parts/kOSMachine1m/part.cfg

```diff
@@ -13,8 +13,8 @@ iconCenter = 0, 3, 0
 
 // --- node definitions ---
 node_attach = 0.0, 0.0, 0.0, 0.0, -1.0, 0.0
-node_stack_bottom = 0.0, -0.14, 0.0, 0.0, -1.0, 0.0
-node_stack_top = 0.0, 0.14, 0.0, 0.0, 1.0, 0.0
+node_stack_bottom = 0.0, -0.173, 0.0, 0.0, -1.0, 0.0
+node_stack_top = 0.0, 0.173, 0.0, 0.0, 1.0, 0.0
```

Comment:
except that if it shows as a diff that means its not in the develop, doesn't it?  did the other pr not get merged right?  at any rate 0.173 is the right number.  Ive been using it a while now.